### ISSUE TITLE
EnableTexCoordBounds breaks pokemon puzzle league video

### ISSUE THREAD

**Issue by fzurita** [gonetz/GLideN64#2496]
When EnableTexCoordBounds is enabled, the Pokemon Puzzle League videos stops working. No need for screenshot, it's just a black screen.

**Comment by fzurita** [comment]
It also breaks starcraft.

**Comment by fzurita** [comment]
@standard-two-simplex In case you haven't seen.

**Comment by standard-two-simplex** [comment]
I can't seem to be able to reproduce this issue, with or without texcoord bounds. No black screen in Pokemon Puzzle League or StarCraft.

**Comment by fzurita** [comment]
Here is what is common between Starcraft 64 and Pokemon Puzzel league:

```glsl_CombinerProgramUniformFactory.cpp:305,ERROR, "bound: tile=0 bounds=(0.000000, 0.000000, 0.000000, 0.000000)"```

In project 64, update is not being called at all somehow.

**Comment by fzurita** [comment]
@standard-two-simplex @gonetz I figured it out why there are differences in behavior between PC and Android. In Android, I don't use the OSD function. When we draw the OSD, we set the drawing state returned by ```dwnd().getDrawer().getDrawingState()``` to ```Non```. This essentially disables the logic ```URasterInfo```.

If you disable the OSD in Project 64, then you will get the same result as Android. This looks like a bug to me.
